seacrowd/sea_datasets/xnli/xnli.py

```diff
@@ -0,0 +1,218 @@
+            description=f"{sub} source schema",
+            schema="source",
+            subset_id=f"{sub}",
+        )
+        for sub in subsets
+    ] + [
+        SEACrowdConfig(
+            name=f"{sub}_seacrowd_pairs",
+            version=datasets.Version(_SEACROWD_VERSION),
+            description=f"{sub} SEACrowd schema",
+            schema="seacrowd_pairs",
+            subset_id=f"{sub}",
+        )
+        for sub in subsets
+    ]
+
+    DEFAULT_CONFIG_NAME = "xnli.vie_source"
+    labels = ["contradiction", "entailment", "neutral"]
+
+    def _info(self) -> datasets.DatasetInfo:
+        if self.config.schema == "source":
+            features = datasets.Features(
+                {
+                    "premise": datasets.Value("string"),
+                    "hypothesis": datasets.Value("string"),
+                    "label": datasets.ClassLabel(names=self.labels),
+                }
+            )
+
+        elif self.config.schema == "seacrowd_pairs":
+            features = schemas.pairs_features(self.labels)
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(
+        self, dl_manager: datasets.DownloadManager
+    ) -> List[datasets.SplitGenerator]:
+        """Returns SplitGenerators."""
+
+        urls = _URLS[_DATASETNAME]
+        data_dir = dl_manager.download_and_extract(urls)
+
+        xnli_train = os.path.join(data_dir["train"], "XNLI-MT-1.0", "multinli")
+        train_data_path = os.path.join(xnli_train, "multinli.train.{}.tsv")
+
+        xnli_test = os.path.join(data_dir["test"], "XNLI-1.0")
+        val_data_path = os.path.join(xnli_test, "xnli.dev.tsv")
+        test_data_path = os.path.join(xnli_test, "xnli.test.tsv")
+
+        lang = self.config.name.split("_")[0].split(".")[-1]
+
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                gen_kwargs={
+                    "filepath": train_data_path,
+                    "split": "train",
+                    "language": _LANGUAGE_MAPPER[lang],
+                },
+            ),
+            datasets.SplitGenerator(
+                name=datasets.Split.VALIDATION,
+                gen_kwargs={
+                    "filepath": val_data_path,
+                    "split": "dev",
+                    "language": _LANGUAGE_MAPPER[lang],
+                },
+            ),
+            datasets.SplitGenerator(
+                name=datasets.Split.TEST,
+                gen_kwargs={
+                    "filepath": test_data_path,
+                    "split": "test",
+                    "language": _LANGUAGE_MAPPER[lang],
+                },
+            ),
+        ]
+
+    def _generate_examples(
+        self, filepath: Path, split: str, language: str
+    ) -> Tuple[int, Dict]:
```

Comment:
change on one-liner based on code-formatter
```suggestion
    def _generate_examples(self, filepath: Path, split: str, language: str) -> Tuple[int, Dict]:
```

Reply:
I used black for formatting, I might need to see somewhere in the documentation about the Seacrowd's standard

Reply:
it uses `black` and `flake8`. you can rune the makefile using this on base dir:

`make check_file=seacrowd/sea_datasets/<dataset_name>/<dataset_name>.py`